test/integration/client-side-encryption/client_side_encryption.prose.test.js

```diff
@@ -1426,6 +1434,291 @@ describe('Client Side Encryption Prose Tests', metadata, function () {
+          // collection with the filter { "encryptedIndexed": <findPayload> }.
+          // Assert less than 10 documents are returned. 0 documents may be returned.
+          // Assert each returned document contains the field
+          // { "encryptedIndexed": "encrypted indexed value" }.
+          const collection = encryptedClient.db('db').collection('explicit_encryption');
+          const result = await collection.find({ encryptedIndexed: findPayload }).toArray();
+          expect(result.length).to.be.below(10);
+          for (const doc of result) {
+            expect(doc.encryptedIndexed).to.equal('encrypted indexed value');
+          }
+        });
+
+        it('returns all documents with contention', async function () {
+          // Use encryptedClient to run a "find" operation on the db.explicit_encryption
+          // collection with the filter { "encryptedIndexed": <findPayload2> }.
+          // Assert 10 documents are returned. Assert each returned document contains the
+          // field { "encryptedIndexed": "encrypted indexed value" }.
+          const collection = encryptedClient.db('db').collection('explicit_encryption');
+          const result = await collection.find({ encryptedIndexed: findPayload2 }).toArray();
+          expect(result.length).to.equal(10);
+          for (const doc of result) {
+            expect(doc.encryptedIndexed).to.equal('encrypted indexed value');
+          }
+        });
+      }
+    );
+
+    context('Case 3: can insert encrypted unindexed', eeMetadata, function () {
+      let insertPayload;
+
+      beforeEach(async function () {
+        // Use clientEncryption to encrypt the value "encrypted unindexed value" with these EncryptOpts:
+        // class EncryptOpts {
+        //    keyId : <key1ID>
+        //    algorithm: "Unindexed"
+        // }
+        // Store the result in insertPayload.
+        insertPayload = await clientEncryption.encrypt('encrypted unindexed value', {
+          keyId: key1Id,
+          algorithm: 'Unindexed'
+        });
+        // Use encryptedClient to insert the document { "_id": 1, "encryptedUnindexed": <insertPayload> }
+        // into db.explicit_encryption.
+        await encryptedClient.db('db').collection('explicit_encryption').insertOne({
+          _id: 1,
+          encryptedIndexed: insertPayload
+        });
+      });
+
+      it('returns unindexed documents', async function () {
+        // Use encryptedClient to run a "find" operation on the db.explicit_encryption
+        // collection with the filter { "_id": 1 }.
+        // Assert one document is returned containing the field
+        // { "encryptedUnindexed": "encrypted unindexed value" }.
+        const collection = encryptedClient.db('db').collection('explicit_encryption');
+        const result = await collection.findOne({ _id: 1 });
+        expect(result.encryptedIndexed).to.equal('encrypted unindexed value');
```

Comment:
needs to assert on `encryptedUnindexed` (also see my note above about using `property` assertions)

Reply:
Updated.